GodotClient/Scenes/Game_Main/Interface/Scripts/Inventory.gd

```diff
@@ -28,19 +37,20 @@ func _process(delta):
 				
 				var slot_pos = get_node(body_part['ident']).get_node('slot0').position
 				
-				var xy = Vector2(slot_pos.x/32,slot_pos.y/32)
-				$inventory_tilemap.set_cellv( xy, item_index )
-				
 				if('opened' in body_part['slot0']):
 					if(body_part['slot0']['opened'] == 'yes'):
 						for item in body_part['slot0']['contained_items']:
-							print(item['ident'])
-							
-							if(startx > 352):
+							if(startx >= 320):
 								startx = 0
 								starty = starty + 32
+							if(starty >= 64):
```

Comment:
Magic numbers.  Can you make these constants and give them meaningful names instead?